### DISCUSSION TITLE
NodePort为什么不能被外部访问？

### DISCUSSION THREAD

**Discussion by liuyuan22** [guangzhengli/kubenates_workshop#18] [Q&A]
在[https://github.com/guangzhengli/k8s-tutorials#nodeport](url)一节中，通过kubectl apply创建了NodePort，但是使用curl命令并没有得到响应，curl命令的ip已经替换成我本地minikube ip命令输出的ip地址，本地的集群是使用minikube创建的，请问这是为什么呢？

**Comment by guangzhengli** [comment]
可以参考这个 Issue https://github.com/guangzhengli/k8s-tutorials/issues/5
